contracts/fixed_price.scilla

```diff
@@ -425,9 +438,13 @@ end
 procedure RequireAccessToCancel(maker_address: ByStr20)
   cur_owner <- contract_owner;
   is_contract_owner = builtin eq cur_owner _sender;
+  cur_operator <- operator;
+  is_operator = builtin eq cur_operator _sender;
   is_maker = builtin eq maker_address _sender;
+  is_maker_owner = orb is_maker is_contract_owner;
+  is_operator_owner = orb is_maker is_operator;
```

Comment:
@razill I think you can simplify the comparison
is_maker_owner = orb is_maker is_contract_owner;
//remove line 445
is_allowed_to_cancel_order = orb is_maker_owner is_operator

Reply:
@inamirandaz We are comparing maker, owner & operator, will have to check maker & owner, maker & operator, and the result is the above.